code/modules/1713/apparel_worldwars.dm

```diff
@@ -1641,6 +1641,13 @@ obj/item/clothing/accessory/storage/webbing/ww1/ww2/stormgroup/Scout
 		new /obj/item/ammo_magazine/rpd(hold)
 		new /obj/item/ammo_magazine/rpd(hold)
 		new /obj/item/weapon/grenade/coldwar/rgd5(hold)
+/obj/item/clothing/accessory/storage/webbing/russian/guns/ak47
```

Comment:
This is already a webbing iirc

Reply:
diff textures so no its not already a webbing so shut